gst/tensor_ros_src/tensor_ros_src.cc

```diff
@@ -32,7 +32,8 @@
 #  include <config.h>
 #endif
 
-#include <tensor_typedef.h>
+#include <nnstreamer/tensor_typedef.h>
+#include <nnstreamer/nnstreamer_plugin_api.h>
```

Comment:
In fact.. both representations work for now.

```<nnstreamer/*.h>``` is supported by pkgconfig and ```<*.h>``` is supported by default ${includedir}.

Reply:
After talking with @wooksong, he decide to use the second one.
* `Use #include <nnstreamer/tensor_typedef.h>` instead of `#include <tensor_typedef.h>`